rumqttc/Cargo.toml

```diff
@@ -18,7 +18,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 default = ["use-rustls"]
 use-rustls = ["dep:tokio-rustls", "dep:rustls-webpki", "dep:rustls-pemfile", "dep:rustls-native-certs"]
-use-native-tls = ["dep:tokio-native-tls", "dep:native-tls"]
+use-native-tls = ["dep:tokio-native-tls", "dep:native-tls", "async-tungstenite/tokio-native-tls"]
```

Comment:
`async-tungstenite/tokio-native-tls` is required by `websocket` ... may I introduce `websocket-native-tls` for native-tls support to avoid such dependency in `use-native-tls`?

Reply:
That's a bad idea, different feature may conflict, forget it.

Reply:
Just saw previous discussion in https://github.com/bytebeamio/rumqtt/pull/742, `?` seems to be good.